OHIF/Viewers
# Issue 793: [TEST E2E] Drag and drop thumbnail into viewport

**Created custom functions:**
- drag and drop: `drag`
- open study: `openStudy`

**Usage:**
- drag and drop: `cy.get('to-drag-element').drag('drop-element')`
- open study: `cy.openStudy('patientName');`

**Test cases implemented:**
- Drag and drop thumbnail into the viewport
- Verify if thumbnails are being displayed
- Verify if side panels Series and Measurements are being displayed/hidden

https://github.com/OHIF/Viewers/issues/793
